### DISCUSSION TITLE
第4节课笔记(1班)

### DISCUSSION THREAD

**Discussion by AXYZdong** [InternLM/Tutorial#267] [笔记]
请在下方提交第 4 节课笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by UncleLLD** [comment]
https://blog.csdn.net/uncle_ll/article/details/135514362

**Comment by woxinyonghen** [comment]
https://blog.csdn.net/xingchengyonghen/article/details/135532187

**Comment by Gu-Xiao** [comment]
https://zhuanlan.zhihu.com/p/677154456

**Reply by Gu-Xiao** [reply]
进阶作业持续进行中

**Comment by biscuit279** [comment]
https://blog.csdn.net/qq_46212981/article/details/135535924

**Comment by xujinzh** [comment]
https://xujinzh.github.io/2024/01/12/ai-internlm-xtuner-finetune/

**Comment by 10086zgyd** [comment]
https://blog.csdn.net/weixin_55546799/article/details/135558024

**Comment by ustbzgn** [comment]
https://blog.csdn.net/qq_46052444/article/details/135566811?spm=1001.2014.3001.5502

**Comment by gpt-dance** [comment]
https://zhuanlan.zhihu.com/p/677614827

**Comment by zpc-666** [comment]
https://blog.csdn.net/zpc1219/article/details/135571472

**Comment by strong-leaf** [comment]
https://zhuanlan.zhihu.com/p/677546293

**Comment by AXYZdong** [comment]
https://nuly9zxzf1.feishu.cn/docx/B4rFdjWMuoiQXwxqqVuc5waynlb#AZ2mdN2tVozEHjxAhhZcMdRNndh

**Comment by Xpg74138** [comment]
https://blog.csdn.net/qq_41776136/article/details/135511608

**Comment by grey2818** [comment]
https://zhuanlan.zhihu.com/p/677977573

**Comment by raulming** [comment]
https://www.notion.so/a2e4e69a582d46e49fd2f1ffe11f1887

**Comment by blurmemo** [comment]
https://github.com/blurmemo/LLM/blob/main/note/InternLM/chap3.md

**Comment by xiaozhuang223** [comment]
https://zhuanlan.zhihu.com/p/677611120
